frontend/src/views/Environments/EnvironmentEditForm.js

```diff
@@ -72,11 +74,16 @@ const EnvironmentEditForm = (props) => {
             tags: values.tags,
             description: values.description,
             dashboardsEnabled: values.dashboardsEnabled,
-            notebooksEnabled: values.notebooksEnabled,
             mlStudiosEnabled: values.mlStudiosEnabled,
             pipelinesEnabled: values.pipelinesEnabled,
             warehousesEnabled: values.warehousesEnabled,
-            resourcePrefix: values.resourcePrefix
+            resourcePrefix: values.resourcePrefix,
+            parameters: [
+              {
+                key: "notebooksEnabled",
+                value: String(values.notebooksEnabled)
+              }
+            ]
           }
```

Comment:
In an ideal world, the frontend would also be modular and we would not need to hardcode "notebooksEnabled" but that is something to think about when we start with the frontend